src/Tribe/Main.php

```diff
@@ -330,7 +330,10 @@ public function hooks() {
 		add_action( 'tribe_help_pre_get_sections', array( $this, 'add_help_section_extra_content' ) );
 		add_filter( 'tribe_support_registered_template_systems', array( $this, 'add_template_updates_check' ) );
 		add_action( 'plugins_loaded', array( 'Tribe__Support', 'getInstance' ) );
-		add_action( 'tribe_events_single_event_after_the_meta', array( $this, 'add_linking_archor' ), 5 );
+
+
```

Comment:
It'd be sweet if you got rid of one of these line breaks so that there's only a single line between chunks.